### PR TITLE
tjlane/refactor

### PR DESCRIPTION
**Pull request by alisiafadini**
Temporarily removed Qweights from itTV + fixed Nbg bug

### CODE REVIEW

**Comment by alisiafadini** [comment]
@tjlane please review!

**Comment by alisiafadini** [comment]
this PR supersedes #1

File: setup.py [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 METEOR contains commandline scripts and functions for finding low occupancy species in crystallographic maps.
 This module will be part of reciprocalspaceship and its "booster" package, rs-booster.
 """
 
-
 try:
-	from setuptools import setup
+	from setuptools import setup, find_packages
 
 except ImportError:
```
Comment (tjlane) [inline setup.py:12]: disutils is deprecated, remove

**Review by tjlane — APPROVED** [review]
lgtm

**Pull request merged**